CHANGELOG.md

```diff
@@ -429,3 +429,4 @@
 * Custom Embargo: Removing Custom Embargo will not enable Save button (UIEH-668)
 * Make eholdings UI permissions effective (UIEH-669)
 * Tags: assign/unassign tags on packages regardless of isSelected value (UIEH-670)
+* Jest+RTL > Test UsageConsolidationContentResource (UIEH-1277)
```

Comment:
Looks like you accidentally added this to old version - we're adding newest version to the top of changelog, and older version are at the bottom
Current WIP version is 7.2.0

Reply:
updated as per the comment.